### DISCUSSION TITLE
Привет!

### DISCUSSION THREAD

**Discussion by MarinaDVetrova** [MarinaDVetrova/Summertime#4] [Q&A] [answered]
Как тебе мой марафон? Присоединишься?

**Comment by HelenaVyzhlets** [comment, marked as answer]
Привет! Когда старт?

**Reply by MarinaDVetrova** [reply]
Старт 1 июля. Самая жара! 🔥 🔥 🔥

**Reply by HelenaVyzhlets** [reply]
Супер!

**Comment by MarinaDVetrova** [comment]
Есть предложения по типам графиков или языкам/библиотекам?

**Reply by HelenaVyzhlets** [reply]
Python, R. От простых до гео. Хочу что-то красивое :)
